openseadragon-filtering.js

```diff
@@ -273,14 +273,18 @@
                 throw new Error(
                     'Brightness adjustment must be between -255 and 255.');
             }
+            var precomputed_BRIGHTNESS = []
```

Comment:
Could you name the variable `precomputedBrightness` instead? (And change for the other filters as well).